src/Cake.Svn/SvnSettings.cs

```diff
@@ -5,5 +5,31 @@
     /// </summary>
     public abstract class SvnSettings
     {
+        /// <summary>
+        /// Ctor.
+        /// </summary>
+        public SvnSettings()
+        {
+            ThrowOnCancel = true;
+            ThrowOnError = true;
+        }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if the 
+        /// operation is canceled. Defaults to true.
+        /// </summary>
+        public bool ThrowOnCancel { get; set; }
+
+        /// <summary>
+        /// Gets or sets a boolean indicating whether the call must throw an error if an
```

Comment:
Should be `Gets or sets a value indicating...`

Reply:
fixed.